Desktop version design

davidobi opened the pull request.

prolajumokeoni commented:
> _No description provided._

- Kindly add a Pull Request message it summarizes changes introduced in more detail in your pull request.

prolajumokeoni reviewed (commented):
Hi @davidobi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a pull request message

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

gemmen29 commented on styles.css:
```diff
@@ -464,3 +456,227 @@ footer .contact-text {
 .submit-contact:hover { font-weight: bold; }
+
+/* DESKTOP VERSION CSS */
+@media only screen and (min-width: 600px) {
```
Please use media query at any point between `992px` and `1100px`
for example:
```
@media screen and (min-width: 990px) {
/*
.
.
*/
}
```

davidobi replied:
Changes have been implemented thanks

gemmen29 commented on index.html:
```diff
@@ -1,260 +1,367 @@
+        <title>David's Portfolio - Mobile</title>
+    </head>
+    <body>
+        <div>
+            <!-- start of desktop nav -->
+            <nav class="desktop-nav">
+                <div class="desktop-logo">
+                    <a href="#" target="_blank" rel="noopener noreferrer">
+                        Welcome
+                    </a>
                 </div>
```
Please try to match the Figma design as much as possible at any point between `992px` and `1100px`
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-davidobi-Portfolio-mobile-version-desktop-version-design-index-html-2022-02-17-14_26_56](https://user-images.githubusercontent.com/24703171/154482772-3c851de5-2f95-4942-97bd-5899493fc4ab.png) | ![Template 2-Main](https://user-images.githubusercontent.com/24703171/154482988-f39d07fc-89fe-48e3-a34d-16871ce3c3e8.png) |

gemmen29 reviewed (changes requested):
Hi @davidobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

davidobi commented on index.html:
```diff
@@ -1,260 +1,367 @@
+    </head>
+    <body>
+        <div>
+            <!-- start of desktop nav -->
+            <nav class="desktop-nav">
+                <div class="desktop-logo">
+                    <a href="#" target="_blank" rel="noopener noreferrer">
+                        Welcome
```
Corrected thanks

Whoistolu replied:
Great implementation 👍🏽

Whoistolu reviewed (changes requested):
Hi @davidobi,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

davidobi commented:
I dont agree with this review truly. this is not fair

davidobi commented:
this is the word of the previous reviewer
Please use media query at any point between 992px and 1100px
for example:

@media screen and (min-width: 990px) {
/*
.
.
*/
}

He said at any point. He didn't say between

Whoistolu commented on styles.css:
```diff
@@ -457,8 +457,234 @@ footer .contact-text {
-/* DESKTOP VERSION CSS */
-@media only screen and (min-width: 600px) {
+/* DESKTOP VERSION CSS LARGE SCREEN SIZE */
+@media only screen and (min-width: 700px) {
```
Hi @davidobi , Kindly implement the changes and you will be approved on the next submission

davidobi commented:
> > this is the word of the previous reviewer Please use media query at any point between 992px and 1100px for example:
> > @media screen and (min-width: 990px) { /* . . */ }
> > He said at any point.
> 
> Hi @davidobi You actually used 600Px and removed it then implemented 700px. ![Screenshot from 2022-02-17 16-51-57](https://user-images.githubusercontent.com/58692590/154518885-89c99102-f737-405a-a035-9bbc400af7fb.png) Please correct it and it will be approved on the next review.

No I just checked the code, you didn't look at my last commit. in it if checked my CSS file I corrected that and added 2 more media queries to account for all screen sizes not just what is required

davidobi commented:
don't worry I would request another review. You could have just saved me the stress if only you actually opened my style.css code that was last pushed

AnselemOdims reviewed (approved):
### Hi  @davidobi 🙋‍♀️, 
Good job in making the required changes 👏 
But please be nicer when talking to a reviewer or your peers.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
